stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -218,182 +207,83 @@ export.load_h_s2_and_product.4
         push.M u32lt assert
         push.M u32lt assert
 
+        horner_eval_base
+
         hperm
     end
 
-    # 3) Load saved claimed hash of h and compare
+    # 4) Load PK, the saved claimed hash of h, and compare
     loc_loadw.0
     movup.4 assert_eq
     movup.3 assert_eq
     movup.2 assert_eq
     assert_eq
```

Comment:
This could be replaced with `assert_eqw`, right? (wouldn't change the cycle count though)

Reply:
Indeed, changed it now